### ISSUE TITLE
Server crashes when crafting high amount of IC2 overclockers (and other random items)

### ISSUE THREAD

**Issue by lokep001** [tth05/morerefinedstorage#50]
#### Issue description:
Server crashes when I request autocraft of high amount of IC2 overclockers. It also crashed when someone on server autocrafted planks? It crashes as soon as you press the button after typing how many you want. (Pre-calculation screen).

Interestingly it only happens on the dedicated server. It has 10GB ram available so I don't really know.
#### What you expected to happen:
It would go on the calculation screen and we can click start.
#### Steps to reproduce:
1. Craft many items on a server.
2. Watch it burn.
3.
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 2860
- Refined Storage: 2.3.2

Does this issue occur on a server? [yes/no]
Yes, only on server
#### If a (crash)log is relevant for this issue, link it here:
For IC2 Overclockers:
https://mclo.gs/N2aLqDq

For Wooden planks (I believe):
https://mclo.gs/ZN4STrz
[pastebin/gist/etc link here]

**Comment by lokep001** [comment]
Thought it mightve been because i forgot to put fluid disks in. but it still happens --> https://mclo.gs/lKQJRtk#L11

**Comment by Pelotrio** [comment]
Rip I changed some things there in my last pr. Will probably check it out tomorrow.

**Comment by lokep001** [comment]
Thanks :D

**Comment by lokep001** [comment]
Yeah noticed it doesn't crash when i removed all external storages for fluid. Hope that helps.

**Comment by Pelotrio** [comment]
> Yeah noticed it doesn't crash when i removed all external storages for fluid. Hope that helps.

Yes thats what I mean I changed the code for external storages in my pr to support longs so that my overloaded integration works. Everything I've tested so far worked with it but apparently I missed some.

**Comment by Pelotrio** [comment]
First issue is fixed in latest commit we are not quite sure about the second one yet. Will probably make a new release after that. You can ask me on discord if you want the version with just the current fix.

**Comment by lokep001** [comment]
So the first issue is the suggestion right?
